enterprise_access/apps/api_client/tests/test_lms_client.py

```diff
@@ -15,17 +15,58 @@ class TestLmsApiClient(TestCase):
     Test LMS Api client.
     """
 
+    @mock.patch('requests.Response.json')
     @mock.patch('enterprise_access.apps.api_client.base_oauth.OAuthAPIClient')
-    def test_get_enterprise_customer_user_data(self, mock_oauth_client):
+    def test_get_enterprise_customer_user_data(self, mock_oauth_client, mock_json):
```

Comment:
wanted to beef this test up with something a little more concrete now that we modify the response json in the lms_client a bit before returning